src/utils.c

```diff
@@ -64,7 +64,6 @@ int shim_signal_table[] = {
 	SIGIO,               /* I/O now possible */
 	SIGPWR,              /* Power failure restart */
 	//SIGSYS,            /* Bad system call */
-	SIGUNUSED,
```

Comment:
How about `#ifdef`-ing this instead of simply removing?

Reply:
for if-deffing, +1 - we still want to work as we did for older versions of the lib/compiler I would think...

Reply:
@jodh-intel I have added the #ifdef to include the signal constant if defined.